commons/src/main/java/org/dhis2/commons/dialogs/MediaDialog.kt

```diff
@@ -0,0 +1,341 @@
+
+                    Text(
+                        text = subTitle,
+                        fontSize = 12.sp,
+                        textAlign = TextAlign.Center,
+                        modifier = Modifier.fillMaxWidth(),
+                        color = colorResource(id = R.color.colorPrimary_f57)
+                    )
+                    Spacer(modifier = Modifier.height(16.dp))
+
+                    LazyColumn(modifier = Modifier.fillMaxWidth()) {
+
+                        for (videoUrl in listOfVideosUrl) {
+                            item {
+                                MediaDialogItem(
+                                    type = MediaType.VIDEO,
+                                    title = "Como lavar as mãos", //Todo: use object field instead
+                                    url = videoUrl,
+                                    duration = generateRandomDuration(), //Todo: use object field instead
+                                    onClickMediaItem = { url -> onMediaItemClicked.invoke(url) },
+                                    dateOfLastUpdate = generateRandomDate() //Todo: use object field instead
+                                )
+                            }
+                        }
+
+                        for (audioUrl in listOfAudioUrl) {
+                            item {
+                                MediaDialogItem(
+                                    type = MediaType.AUDIO,
+                                    title = "Como lavar as mãos", //Todo: use object field instead
+                                    url = audioUrl,
+                                    duration = generateRandomDuration(), //Todo: use object field instead
+                                    onClickMediaItem = { url -> onMediaItemClicked.invoke(url) },
+                                    dateOfLastUpdate = generateRandomDate() //Todo: use object field instead
+                                )
+                            }
+                        }
+                    }
+
+                    ButtonClose(
+                        modifier = Modifier.align(Alignment.End),
+                        onClick = { onDismiss.invoke() }
+                    )
+                }
+            }
+        }
+    }
+}
+
+@Composable
+private fun ButtonClose(
+    modifier: Modifier,
+    onClick: () -> Unit,
+) {
+    OutlinedButton(
+        onClick = onClick,
+        border = BorderStroke(0.5.dp, colorResource(R.color.colorPrimary_b0b)),
+        shape = RoundedCornerShape(50),
+        modifier = modifier
+    ) {
+        Text(
+            text = stringResource(id = R.string.media_dialog_label_close),
+            fontSize = 12.sp,
+            color = colorResource(R.color.textPrimary)
+        )
+    }
+}
+
+@Composable
+private fun MediaData(
+    title: String,
+    url: String,
+    duration: String,
+    dateOfLastUpdate: String,
+    onClickMediaItem: (url: String) -> Unit,
+) {
+    val textSize = remember { 12.sp }
```

Comment:
you don't need to use remember to assign and you should move default text sizes to typo file